### PR TITLE
Added always run option to distroless test

### PR DESCRIPTION
**Pull request by utka**

### CODE REVIEW

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml [hunk 1]
```diff
@@ -455,6 +455,7 @@ presubmits:
         testing: test-pool
   - name: e2e-simpleTests-distroless-master
     <<: *job_template
+    always_run: true
```
Comment (howardjohn) [inline prow/cluster/jobs/istio/istio/istio.istio.master.yaml:458]: can we add `skip_report: true` until https://github.com/istio/istio/issues/15414 is fixed?

**Review by fejta — APPROVED** [review]
/hold

**Review by fejta — APPROVED** [review]

**Review by fejta — COMMENTED** [review]
/lgtm

**Comment by howardjohn** [comment]
Fixes https://github.com/istio/test-infra/issues/1418

File: prow/cluster/jobs/istio/istio/istio.istio.master.yaml (hunk 1 above)
Comment (howardjohn) [reply prow/cluster/jobs/istio/istio/istio.istio.master.yaml:458]: Fixed

Comment (howardjohn) [reply prow/cluster/jobs/istio/istio/istio.istio.master.yaml:458]: https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio/15608/e2e-simpleTests-distroless-master/3

**Review by howardjohn — APPROVED** [review]

**Pull request merged**